package.json

```diff
@@ -231,7 +233,8 @@
     "elliptic": "6.6.1",
     "graphql-request": "^6.1.0",
     "json5": "^2.2.2",
-    "follow-redirects": "^1.15.4"
+    "follow-redirects": "^1.15.4",
+    "solid-js": "^1.9.4"
```

Comment:
audit issues can be fixed with fewer resolutions, pls see https://github.com/madfish-solutions/templewallet-extension/pull/1272/commits/0618fcaae57e1d238ae991c5844d87c8c80eb4ac

Reply:
Sorry, it's not the case because lots of additional modules relying on `axios "1.7.7"` have been installed to work with Ledger EVM accounts

Reply:
yeah, but you can still get rid of solid-js resolution, just edit yarn.lock